ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateSymbolStylesFromWebStyles/CreateSymbolStylesFromWebStyles.qml

```diff
@@ -0,0 +1,201 @@
+                                                                       return;
+
+                                                                       legendItems = symbolStyle.searchSymbolsResult;
+                                                                       symbolStyle.searchSymbolsResult.forEach((symbolResult) => {
+                                                                                                                   symbolResult.fetchSymbolStatusChanged.connect(() => {
+                                                                                                                                                                     if (symbolResult.fetchSymbolStatus !== Enums.TaskStatusCompleted)
+                                                                                                                                                                     return;
+
+                                                                                                                                                                     const label = symbolResult.key;
+                                                                                                                                                                     const values = getValuesFromSymbolLabel(label);
+                                                                                                                                                                     const symbol = symbolResult.fetchSymbolResult;
+                                                                                                                                                                     values.forEach((value) => {
+                                                                                                                                                                                        const uniqueValue = ArcGISRuntimeEnvironment.createObject("UniqueValue", {label: label, values: [value], symbol: symbol}, webLayer);
+                                                                                                                                                                                        uniqueValueRenderer.uniqueValues.append(uniqueValue);
+                                                                                                                                                                                    });
+                                                                                                                                                                 });
+                                                                                                                   symbolResult.fetchSymbol();
+                                                                                                               });
```

Comment:
This is pretty gnarly to read...

In QML connect statements, you can also just pass in a JS function. It might be better to do that than to have several nested arrow functions.

We do this in other samples, for example here we are connecting to a JS function that is created a few lines below it. It can help keep things a bit more tidy and readable. https://github.com/Esri/arcgis-runtime-samples-qt/blob/master/ArcGISRuntimeSDKQt_QMLSamples/Maps/GenerateOfflineMap/GenerateOfflineMap.qml#L116